tox.ini

```diff
@@ -19,8 +19,7 @@ setenv =
     XDG_CACHE_HOME = {envdir}
     SKIP_GENERATE_AUTHORS = 1
     SKIP_WRITE_GIT_CHANGELOG = 1
-envars =
-    PATH = {env:PATH}:{toxinidir}/bin
+    PYTHONPATH = {env:PYTHONPATH}:{toxinidir}/server/lib:{toxinidir}/lib
```

Comment:
PYTHONPATH seems to work only in setenv

Reply:
I think you don't understand.  `envars` is not a recognized configuration option, so PyTest simply ignores it and its values; however, by removing (old) lines 22-23, the `PYTHONPATH` key moves from `envars` to the `setenv` option, which makes it meaningful.

But, at the moment, the `PYTHONPATH` definition is attached to the `setenv` option under the `[testenv]` section.  Options in this section are applied to all `[testenv:*]` sections, which is undesirable, since part of your change applies only to the Server.  So, you should move the server-specific part of change under the `[testenv:server]` section:
```
[testenv]
...
setenv =
    ...
    PYTHONPATH = {env:PYTHONPATH}:{toxinidir}/lib
...
[testenv:server]
...
setenv =
    PYTHONPATH = {env:PYTHONPATH}:{toxinidir}/server/lib
```

Reply:
Oh, ok got it.